sippy-ng/src/component_readiness/ReleaseSelector.js

```diff
@@ -20,30 +39,65 @@ const useStyles = makeStyles((theme) => ({
 
 function ReleaseSelector(props) {
   const classes = useStyles()
-  const [versions, setVersions] = React.useState([])
+  const [versions, setVersions] = React.useState({})
   const { label, version, onChange } = props
```

Comment:
You can stuff the 4 new parameters in here.  If you do that, you can eliminate the `props.` from the references below.
```suggestion
  const { label, version, onChange, startTime, setStartTime, endTime, setEndTime, } = props
```

Reply:
Sure will fix since that's the style in here. I personally prefer the `props.` style since I know if the vars are local or properties